docs/openapi/info-pa-schemas/info-pa.yaml

```diff
@@ -54,4 +54,10 @@ components:
           description: The title of the product
         productUrl:
           type: string
-          description: The URL of the product
\ No newline at end of file
+          description: The URL of the product
+    
+    BffPaGroupStatus:
+      $ref: 'https://raw.githubusercontent.com/pagopa/pn-external-registries/f49e1a815e81a70834c21c95c0fb7e9e8d39c054/docs/openapi/pn-selfcare-internal-v1.yaml#/components/schemas/PaGroupStatus'
+    
+    BffPaGroupList:
```

Comment:
Check on front-end side if we use all the data from PaGroupList. We should use only the name and the id.
If true, define BffPaGroupList as an array of BffPaGroup where BffPaGroup has only those properties that we use on the front-end side.